src/components/chip-clickable/chip-clickable.tsx

```diff
@@ -0,0 +1,119 @@
+import { Component, Host, h, Prop, Event, EventEmitter, Element, State } from '@stencil/core';
+
+export type Color = 'default' | 'info' | 'success' | 'warning' | 'danger' | 'outline';
+export type Size = 'standard' | 'tall';
+
+@Component({
+  tag: 'bds-chip-clickable',
+  styleUrl: 'chip-clickable.scss',
+  shadow: true,
+})
+export class ChipClickable {
+  @Element() private element: HTMLElement;
+  @State() visible = true;
+  /**
+   * used for add icon in left container. Uses the bds-icon component.
+   */
+  @Prop() icon?: string;
+  /**
+   * used for add avatar left container. Uses the bds-avatar component.
+   */
+  @Prop() avatar?: string;
+  /**
+   * used for change the color. Uses one of them.
+   */
+  @Prop() color?: Color = 'default';
+  /**
+   * used for change the size chip. Uses one of them.
+   */
+  @Prop() size?: Size = 'standard';
+  /**
+   * it makes the chip clickable.
+   */
+  @Prop() clickable?: boolean = false;
+  /**
+   * used for delete the chip.
+   */
+  @Prop() close?: boolean = false;
+  /**
+   * the chip gone stay disabled while this prop be true.
+   */
+  @Prop() disabled?: boolean = false;
+  /**
+   *  Triggered after a mouse click on close icon, return id element. Only fired when close is true.
+   */
+  @Event() chipClickableClose: EventEmitter;
+  @Event() chipClickableClick: EventEmitter;
+
+  private handleClick(event) {
+    if (event.key === 'Enter' && !this.disabled) {
+      event.preventDefault();
+      this.chipClickableClick.emit();
+    }
+  }
+  private handleCloseChip(event) {
+    if (event.key === 'Enter' && !this.disabled) {
+      event.preventDefault();
+      this.chipClickableClose.emit({ id: this.element.id });
+      this.visible = false;
```

Comment:
is that correct? both cases set visible to false, if is, you should move this.visible to outside of if

Reply:
Done!